docs/src/main/sphinx/security/group-mapping/file.md

```diff
@@ -16,10 +16,21 @@ file.group-file=/path/to/group.txt
 
 The following configuration properties are available:
 
-| Property              | Description                                           |
-| --------------------- | ----------------------------------------------------- |
-| `file.group-file`     | Path of the group file.                               |
-| `file.refresh-period` | How often to reload the group file. Defaults to `5s`. |
+:::{list-table} File group provider configuration
```

Comment:
I dont think we want to move to a new deeper directory structure just for that one file